net/netfilter/nf_tables_api.c

```diff
@@ -8613,11 +8613,6 @@ static int __nf_tables_abort(struct net *net, enum nfnl_abort_action action)
 		nf_tables_abort_release(trans);
 	}
 
-	if (action == NFNL_ABORT_AUTOLOAD)
-		nf_tables_module_autoload(net);
-	else
-		nf_tables_module_autoload_cleanup(net);
-
```

Comment:
This needs the change from the previous commit.

Reply:
I believe the reordering of code from the fix I just made will also cover this.  Resolving for now.